django_x509/base/admin.py

```diff
@@ -73,6 +74,35 @@ def get_fields(self, request, obj=None):
             fields.remove('passphrase')
         return fields
 
+    def get_context(self, data, ca_count=0, cert_count=0):
+        context = dict()
+        if ca_count:
+            message = _('Are you sure you want to renew the selected CAs and their related certificates? '
+                        'All the CAs listed below with related certificates will be renewed.')
```

Comment:
there's no reason to put this here, can you put it back in the template and use `blocktranslate` if needed please?
use <br> to separate the two lines.